package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "seagull",
-  "version": "0.9.0",
+  "version": "0.10.0-rc.1",
```

Comment:
Is this what you want?

Reply:
I definitely want to go from `0.9.0` to `0.10.0`. I'm just prepping for the release candidate 😉